src/EventManager.php

```diff
@@ -281,7 +282,7 @@ protected function triggerListeners(EventInterface $event, ?callable $callback =
     {
         $name = $event->getName();
 
-        if (empty($name)) {
+        if ($name === null || $name === '') {
```

Comment:
can name be "0" ? see https://getrector.com/demo/66ffa9db-8773-4373-ac55-e91f5a95fcc1

Reply:
The doc block is `string|null` for `getName()` - any non-empty-string should be fine, so `"0"` should be ok but I doubt there's a test covering that.

Reply:
On second thoughts, PHP would cast that to `(int) 0` in the array of event names. Still wouldn't be a problem, but better to preserve existing behaviour.